fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/PartialDataRequestHandler.java

```diff
@@ -111,9 +114,11 @@ public boolean handleRequest(
      * @return the missing intervals from the request or an empty list
      */
     public static SimplifiedIntervalList getPartialIntervalsWithDefault(Map<String, Serializable> context) {
-        return (SimplifiedIntervalList) context.computeIfAbsent(
-                MISSING_INTERVALS_CONTEXT_KEY.getName(),
-                (ignored) -> new SimplifiedIntervalList()
+        return new SimplifiedIntervalList(
```

Comment:
Need a comment here to the effect of 'A serialization issue may result in the context key being a list but not a Simplified Interval List'

And a javadoc with  'fix' section, referencing a link to the issue.

Reply:
@michael-mclawhorn Thanks. I've update this issue: https://github.com/yahoo/fili/issues/657